extensions/amp-sidebar/1.0/component.js

```diff
@@ -215,20 +220,24 @@ export function SidebarToolbar({
 
     const selector = `#${CSS.escape(toolbarTarget)}`;
     const newTargetEl = document.querySelector(selector);
+    console.log('targetEl', newTargetEl, selector);
```

Comment:
How do we know this is the right time to look for the `document.querySelector(selector)`? What if it's not in the DOM yet because it hasn't finished parsing?